Frontend to Main

MlDOR1 opened the pull request:
Pages Created with .jsx and .css

react-router-dom is installed so don't forget to npm install in frontend folder

Rawitath committed:
Create React Pages with Routing

Pages Created with .jsx and .css

react-router-dom is installed so don't forget to npm install in frontend folder

MlDOR1 committed:
Repush(wrong Branches)

MlDOR1 committed:
Added Import React

MlDOR1 committed:
Test

Rawitath reviewed (commented):
Is this accidentally merge?

Rawitath reviewed (changes requested):
Pull Request ตลอดปายยย

asiaapipoo committed:
HMTL and CSS

MlDOR1 committed:
Update index.html

asiaapipoo committed:
Update Frontend แบบฏ้อปวางเปลี่ยนชื่อ

asiaapipoo committed:
Again เปลี่ยนชื่อ

asiaapipoo committed:
again and again

asiaapipoo committed:
zzz

asiaapipoo committed:
Merge branch 'frontend' of https://github.com/Rawitath/We-Will-Cook into frontend

MlDOR1 committed:
Organized Pages and Modified login register page

Added "Already have an account? Login" button on the bottom

asiaapipoo committed:
กุลืม commit TT

asiaapipoo committed:
WTF

MlDOR1 committed:
Updated Register

MlDOR1 committed:
Added back button

Added back button to the box

MlDOR1 committed:
NPM ver18

MlDOR1 committed:
added index.html to public folder due to error

asiaapipoo committed:
forgot แปะไว้เฉยๆยังไม่ทำ

MlDOR1 committed:
Main Webpage มั้ง ในที่สุด

MlDOR1 committed:
Updated website

MlDOR1 committed:
Updated dark mode into webpage

asiaapipoo committed:
.

asiaapipoo committed:
ทำให้หมดละนะ fg pass

อยากให้ปรับไรก้บอก

asiaapipoo committed:
logo login

MlDOR1 committed:
Polished Webpage design

MlDOR1 committed:
Merge branch 'frontend' of https://github.com/Rawitath/We-Will-Cook into frontend

asiaapipoo committed:
จัดไปน้องชาย

MlDOR1 committed:
Revamped landing page to match the sketch.

removed the taste peference buttons replaced with words to encourage user to create an account and also let user use guest mode

MlDOR1 committed:
Bruh

MlDOR1 committed:
Added noodle customization pages.

MlDOR1 committed:
Revamp Noodle customization page.

MlDOR1 committed:
Revamped all the frontend system

MlDOR1 committed:
Updated website.

Added drop down menu and Added summary pages.

MlDOR1 committed:
Added missing pages

MlDOR1 committed:
Updated?????

MlDOR1 committed:
Bruh

MlDOR1 committed:
Merge branch 'main' into frontend

Rawitath reviewed (approved):
LGTM Bro Let's Gooooooooooooooooooooooooo
